src/python/pants/rules/core/lint_test.py

```diff
@@ -0,0 +1,67 @@
+# Copyright 2019 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+from typing import List
+
+from pants.engine.legacy.graph import HydratedTarget, TransitiveHydratedTargets
+from pants.engine.legacy.structs import PythonTargetAdaptor
+from pants.engine.rules import UnionMembership
+from pants.option.option_value_container import OptionValueContainer, RankedValue
+from pants.rules.core.lint import Lint, LintResult, TargetWithSources, lint
+from pants.testutil.engine.util import MockConsole, MockGet, run_rule
+
+
+def test_transitive_option() -> None:
```

Comment:
Especially needing to duplicate an effectively identical test makes duplicating the option a non-starter I think.